src/wavesurfer.js

```diff
@@ -441,6 +441,39 @@ var WaveSurfer = {
     },
 
     /**
+     * Unmounts the wavesurfer instance from the page, but keeps the audio
+     * player in memory
+     */
+    unmount: function (selector) {
+        if (this.container.children.length === 0) {
+            var wsNode = document.querySelectorAll(selector)[0];
```

Comment:
The wavesufer element can be accessed as `this.drawer.wrapper`. It's better to define the unmount method in `drawer.js` and just call it from here.